### PR TITLE
adds ores

### PR DESCRIPTION
**Pull request by utahclock**
addes new ores per issue #332 (do not merge to master code.)

(would merge to mining-rework branch but it wants to add 53 files)
(please add this to mining-rework. or update mining rework so i can add this push properly.)
(ore spawns will need to be adjusted as they are set pretty high for testing purposes)

:cl: utahclock
rscadd: Nanotrasen supply department equipment has been upgraded to detect several new ores. This will make the miners have to labor harder, which NT heads approve. DO WORK!, The ores are detectable on scanners and can be processed for use around the station. /:cl:

### CODE REVIEW

**Comment by ghost** [comment]
at least one of the sprites looks like a literal turd and another looks like bacon and  yet a third grossly old and forgotten bacon

**Comment by DavidEdwards** [comment]
What are you on about Adam. Copper is pretty much turd coloured, so you are pretty much going to have to deal with that.

As for the bacon one, @utahclock you should make the blue bacony shaped one look more like a sheet. The other sprites are usually represented by a flat sheet.

**Comment by FluffySurvivor** [comment]
AMAZING, WE NOW HAVE BACONIUM IN THE GAME.

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -8,6 +8,11 @@ Mineral Sheets
 		- Gold
 		- Silver
 		- Clown
+		- Coal
+		- Platnium
```
Comment (FluffySurvivor) [inline code/game/objects/items/stacks/sheets/mineral.dm]: Platinum*

**Comment by utahclock** [comment]
recolored copper.

**Comment by DavidEdwards** [comment]
Looks better now. I won't merge this PR directly, because it will end up in dev (and therefore eventually into master)

**Comment by FluffySurvivor** [comment]
PLEASE KEEP THE BACON SPRITE 
RENAME IT TO BACONIUM

**Comment by utahclock** [comment]
Yea please dont merge till project is done. Ive got floors and tiles finished thst ill be adding to pr when i csn get back to my computer. Then i grt to work on mineral doors. Tog is going to help with statues. Then have to add adam content. Once thats done and sub floor for mining is added it will be ready for merge. If you csn please add dont merge tag and work in progress to it.

**Comment by FluffySurvivor** [comment]
> still no baconium

I take offense to baconium racism.

**Comment by oisin100** [comment]
@utahclock You should make a new PR on the new repository once this is done.

**Pull request closed without merging**